assets/src/components/customers/CustomerDetailsModal.tsx

```diff
@@ -7,6 +7,38 @@ import {Button, Modal, Paragraph, Text} from '../common';
 // TODO: create date utility methods so we don't have to do this everywhere
 dayjs.extend(utc);
 
+function CustomerMetadataSection({
+  metadata,
+}: {
+  metadata: Record<string, string>;
+}) {
+  if (!metadata) {
+    return null;
+  }
+  return (
+    <React.Fragment>
+      <Box mb={2}>
+        <Box>
+          <Text strong>Custom metadata</Text>
+        </Box>
+      </Box>
+      <Flex sx={{justifyContent: 'space-between', flexWrap: 'wrap'}}>
+        {Object.entries(metadata).map(([title, value]) => {
+          return (
+            <Box mb={2} sx={{flex: '0 50%'}} key={title}>
+              <Box>
+                <Text strong>{title}</Text>
+              </Box>
+
+              <Paragraph>{value as string}</Paragraph>
```

Comment:
just curious, why is the `as string` necessary here?

Reply:
Sorry it was because the types didn't align before, and forgot to edit. The Paragraph didn't want a number for example, which is legal in JSON, so I just forced it to be a string.